tests_e2e/orchestrator/scripts/install-agent

```diff
@@ -85,19 +85,19 @@ echo "Agent Version:"
 $python "$waagent" --version
 echo "Service Status:"
 
-# Sometimes the service can take a while to start; give it a few minutes,
+# Sometimes the service can take a while to start and provision completed; give it a few minutes,
```

Comment:
I'd update the comment to describe that we need to wait for the provisioning code before stopping the agent's service to do the test setup

Reply:
addressed